docs/guide/integrations-guide.md

```diff
@@ -185,13 +185,13 @@ We accept JSON messages using any log shipper and any logging library, as long a
 
 ### Monitoring
 
-When creating a monitoring app two steps are required, package installation and agent configuration setup.
+When creating a monitoring app two steps are required, Agent Installation and Agent Setup.
```

Comment:
When creating a monitoring app two steps are required, Agent Installation and Agent Setup.

=>

Two steps are required when creating a Monitoring App:
1. Agent Installation
2. Agent Setup